docs/eventing/sources/sinkbinding/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/index
+  - /docs/eventing/samples/sinkbinding/README
+  - /docs/eventing/sources/sinkbinding.md
```

Comment:
```suggestion
  - /docs/eventing/samples/sinkbinding/
  - /docs/eventing/sources/sinkbinding
```

I think this would also work but maybe check with @RichieEscarez

Reply:
Yes, we want the site URLs here not the source paths -> exclude the file type suffix

Reply:
Also, these aliases work on staging, so if you are unsure, you can test them in the "Preview" at https://app.netlify.com/sites/knative/deploys